AdyenSession/API/Session Setup/SessionRequest.swift

```diff
@@ -84,5 +84,7 @@ extension SessionSetupResponse {
     internal struct Configuration: Decodable {
         
         internal let installmentOptions: InstallmentConfiguration?
+        
+        internal let enableStoreDetails: Bool
```

Comment:
There is neither class nor protocol named `SessionRequest`. Why renaming?

Reply:
Another good catch, I renamed the wrong file :)